python/dgl/nn/utils.py

```diff
@@ -0,0 +1,80 @@
+"""Graph construction utilities used in dgl.nn modules"""
+# pylint: disable= no-member, arguments-differ
+
+import numpy as np
+
+from ..graph import DGLGraph
+from ..batched_graph import BatchedDGLGraph, batch
+from ..graph_index import from_csr
+from .. import backend as F
+
+
+__all__ = ['_create_fully_connected_graph', '_create_bipartite_graph',
+           '_create_graph_from_num_nodes']
+
+def _create_fully_connected_graph(graph):
+    r""" Create a fully connected graph upon the given graph."""
```

Comment:
What you need is a segment index -- the i^th element specifies how many nodes are there in the result i^th graph. Thus, it's better not use `graph` as the argument.

Reply:
DONE